app/assets/javascripts/admin_users.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
hahaha, try to not commit things you wont use and are just comments, plus coffee script is die